utils/logger.js

```diff
@@ -0,0 +1,109 @@
+const createLogger = () => {
+  const isProdMode =
+    process.env.NODE_ENV === "prod" || process.env.NODE_ENV !== "development"
+      ? true
+      : false;
+
+  //TODO: move this var to env file.
+  // level 1 = debug
+  // level 2 = default
+  // level 3 = no logging
+  let logLevel = 2;
+
+  if (isProdMode) {
+    return createProdLogger(logLevel);
+  }
+
+  return createDevLogger(logLevel);
+};
+
+const createProdLogger = (logLevel) => {
+  //TODO: read log destination from env. (QA/Prod might be different destination, but log the same)
+  //TODO: implement a real Prod logger
+
+  const debugLogFunction = (message) => {
+    if (logLevel > 1) return;
+
+    const currentTime = new Date().toLocaleTimeString();
+    console.log(`${currentTime}: ${message}`);
+  };
+
+  const infoLogFunction = (message) => {
+    if (logLevel > 2) return;
+
+    const currentTime = new Date().toLocaleTimeString();
+    console.log(`${currentTime}: ${message}`);
```

Comment:
Can we prepend the log level (`INFO`/`DEBUG`/`WARN`/`ERROR`) before the message? This helps introspect into the logging we're doing and clarifies the relative importance of log messages at verbose log levels.

Reply:
💯